### DISCUSSION TITLE
Showing created/updated date in collection detail view. Possible?

### DISCUSSION THREAD

**Discussion by simonwidjaja** [strapi/strapi#7965] [Legacy Topics]
Is it possible to show created/updated date in collection detail view? I wasn't able to achieve that without adding another custom date field that will be populated via a hook. But that is not really elegant since it means redundant fields and an unnecessary hook...
Am I missing something? Any hint appreciated.

**Comment by soupette** [comment]
You can click on the cog icon in the right side of the list view just above the table's label. Then, you need to click on the configure the view button and you will access a settings page where you can select the displayed labels of your collection type, reorder them, set the default page limit, sorting options...

Here's the list view
![Screenshot 2020-09-21 at 09 31 22](https://user-images.githubusercontent.com/13311463/93741759-487a0a80-fbed-11ea-9560-3881099cab2f.png)

Here's the setting view
![Screenshot 2020-09-21 at 09 31 36](https://user-images.githubusercontent.com/13311463/93741761-49ab3780-fbed-11ea-8c25-f54ea819d3bd.png)

**Comment by simonwidjaja** [comment]
I need the dates on the **detail page**. In the list view I can show the dates without any problems. I need those data somewhere on the detail view. I think the right hand column (where relations would appear) would be the perfect place to show these meta data. Just like Wordpress is doing it.

If this is not possible out of the box, can I extend the core views of Strapi in any why without "breaking the seal"? I started deep diving into Strapi plugin development last week (which I am super happy with so far), but I didn't find any documentation on extending core view so far...

So any hint is highly appreciated.

Detail Page:
![Strapi2](https://user-images.githubusercontent.com/1757398/93742470-74e25680-fbee-11ea-834d-3664e6709111.jpg)

![strapi_detail](https://user-images.githubusercontent.com/1757398/93742156-ef5ea680-fbed-11ea-9086-e039a5a132c8.jpg)

**Comment by derrickmehaffy** [comment]
@simonwidjaja can you open up a feature request on the issues instead of a discussion?
